dindex_store/profile_index_duckdb.py

```diff
@@ -0,0 +1,55 @@
+import os
+import duckdb
+
+from typing import Dict
+
+from dindex_store.discovery_index import ProfileIndex
+
+
+class ProfileIndexDuckDB(ProfileIndex):
+
+    def __init__(self, config: Dict) -> None:
+        ProfileIndexDuckDB._validate_config(config)
+        self.config = config
+        self.conn = duckdb.connect()
```

Comment:
throughout dindex_store we connect to duckdb and kuzu without specifying a target "path". I know they use a default. I think it's important we pass the path explicitly (through the config file) as at any given time, we may have different dindex_stores available.